pkg/sqlstore/migrations/migrations.go

```diff
@@ -110,3 +111,24 @@ func createAPIKeyTableMigration() *gormigrate.Migration {
 		},
 	}
 }
+
+func createAnnotationsTableMigration() *gormigrate.Migration {
+	type annotation struct {
+		ID        uint   `gorm:"primarykey"`
+		AppName   []byte `gorm:"type:varchar(255);not null;default:null"`
```

Comment:
I thought this would not allow creating empty fields, but it doesn't seem to work:

```
❯ curl -X POST -d '{ "appName": "", "content": "" }' localhost:4040/api/annotations
{"appName":"","content":"","timestamp":1662585565}
```

Reply:
The `annotation` type definition is private and can't be used outside the migration function (this is done so in order to avoid side effects caused by model type changes; this is not mandatory but considered a good practice).

The reason why API allows creating an annotation with empty fields is that it uses `model.Annotation` which does not have constraints:
```go
type Annotation struct {
 	AppName   string
 	Content   string
 	Timestamp time.Time
 	CreatedAt time.Time
 	UpdatedAt time.Time
 }
```